protostar/starknet/cheatcode.py

```diff
@@ -20,6 +20,8 @@
 
 
 class Cheatcode(BusinessLogicSysCallHandler, HintLocal):
+    _exec_locals = {}
+
```

Comment:
- Why is this a static property of Cheatcode? Only `ReflectCheatcode` uses it.
- This property uses private property notation, but it's used as a public property.
- A comment explaining this property would be useful.

Reply:
1. It allows all future cheatcodes to use execution locals and they can be very useful.
2, 3. Good points 👍

Reply:
https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it